README.md

```diff
@@ -117,6 +117,15 @@ Otherwise it is a 32 byte hash.
 
 Boolean whether archive is live. `true` by default. Note that its only populated after archive.open(cb) has been fired.
 
+#### `archive.version`
+
+The label of the most recent checkpoint.
+
+#### `archive.semver`
+
+The label of the most recent semantic-version checkpoint.
```

Comment:
the semver thing sounds like an addition that could be useful too but might better be discussed in a separate pull request?